### ISSUE TITLE
`unison init` suggestion from UCM should match executable name

### ISSUE THREAD

**Issue by rlmark** [unisonweb/unison#2152]
Many folks use an install of the codebase manager where the executable is known as `ucm`. When prompting the user to create a codebase when not found, the prompt suggests to run `unison init,` which may be confusing for folks if their codebase manager executable is named differently.

**Comment by runarorama** [comment]
Good catch. But also we should just init the codebase in the default location without asking the user to jump through hoops.
